src/LayerPlan.cpp

```diff
@@ -849,8 +849,13 @@ void LayerPlan::writeGCode(GCodeExport& gcode)
             if (path.config->isTravelPath())
                 speed *= extruder_plan.getTravelSpeedFactor();
             else
+            {
                 speed *= extruder_plan.getExtrudeSpeedFactor();
 
+                // for some movements such as prime tower purge, the speed may get changed by this factor
+                speed *= path.config->getExtrusionSpeedFactor();
```

Comment:
We discussed this in person and I said we could have just as well made the factor work for travel moves as well, but it's not necessary for you to fix this now. We can also do this in the future.